### PR TITLE
Feature/dp 676 beta feedback banner

### PR DESCRIPTION
**Pull request by shilpigoel-goaco**

### CODE REVIEW

File: Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml [hunk 1]
```diff
@@ -130,10 +130,10 @@
         <div class="govuk-phase-banner">
             <p class="govuk-phase-banner__content">
                 <strong class="govuk-tag govuk-phase-banner__content__tag">
-                    Prototype
+                    Beta
                 </strong>
                 <span class="govuk-phase-banner__text">
-                    This is a prototype of a service in development.
+                    <a class="govuk-link" target="_blank" href="/provide-feedback-and-contact/?context=feedback">Contact the Find a Tender team</a> if you have feedback, questions or suggestions.
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml:136]: Links that open in a new tab need to be marked as such. As in the following example from https://design-system.service.gov.uk/components/phase-banner/#add-a-feedback-link

{3DF1029B-C4ED-4D08-9671-45BDB7D2D0AD}

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+@page "/confirmationmessage"
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml:1]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+                }
+
+                <div class="govuk-form-group govuk-!-margin-bottom-6">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+                            <span class="govuk-fieldset__heading">What's it to do with?</span>
+                        </legend>
+
+                        <div class="govuk-radios" data-module="govuk-radios" data-govuk-radios-module-started="true">
+                            <div class="gem-c-radio govuk-radios__item">
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:46]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+
+<main id="content" role="main">
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <div class="gem-c-title govuk-!-margin-top-8 govuk-!-margin-bottom-8">
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml:6]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+@page "/confirmationmessage"
+
+<main id="content" role="main">
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ConfirmationMessage.cshtml:3]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml (hunk 1 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/Shared/_Layout.cshtml:136]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 6]
```diff
@@ -0,0 +1,122 @@
...
+
+                        <div class="govuk-form-group">
+
+                            <label for="email" class="gem-c-label govuk-label">Your email address</label>
+
+                            <div class="gem-c-hint govuk-hint">
+                                We'll only use this to reply to your message.
+                            </div>
+
+                            <input aria-describedby="hint-a28138ec" autocomplete="email" class="gem-c-input govuk-input" id="@nameof(Model.Email)" name="@nameof(Model.Email)" spellcheck="false" type="email" value="@(Model.Email)">
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:111]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 7]
```diff
@@ -0,0 +1,122 @@
...
+
+                            <div class="gem-c-radio govuk-radios__item">
+                                <input type="radio" name="feedbackOrContactOption" id="SpecificPage" value="SpecificPage" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "SpecificPage" ? "checked" : "") aria-controls="conditional-SpecificPage" aria-expanded="true">
+                                <label for="location-1" class="gem-c-label govuk-label govuk-radios__label">A specific page</label>
+                            </div>
+
+                            <div class="govuk-radios__conditional govuk-body govuk-radios__conditional--hidden" id="conditional-SpecificPage">
+                                <div class="govuk-form-group">
+                                    <label for="link" class="gem-c-label govuk-label">Enter URL or name of page</label>
+                                    <input class="gem-c-input govuk-input" data-ga4-form-include-input="" id="UrlOfPage" name="UrlOfPage" spellcheck="false" type="text" value="@Model.UrlOfPage">
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:59]: Clever, I wasn't sure how to do that. Done.

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml.cs [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+using Microsoft.IdentityModel.Tokens;
+using System.ComponentModel.DataAnnotations;
+
+
+[AuthenticatedSessionNotRequired]
+public class ProvideFeedbackAndContact(IOrganisationClient organisationClient) : PageModel
+{
+    [BindProperty]
+    [DisplayName("What's it to do with")]
+    [Required(ErrorMessage = "Choose a option")]
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml.cs:19]: Done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml.cs [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+            return Page();
+        }
+
+        if ((!string.IsNullOrEmpty(ModelState[nameof(Email)]!.RawValue as string)) &&
+            (ModelState[nameof(Name)]!.RawValue as string == string.Empty))
+        {
+            ModelState[nameof(Name)]!.ValidationState = Microsoft.AspNetCore.Mvc.ModelBinding.ModelValidationState.Invalid;
+            ModelState[nameof(Name)]!.Errors.Add("Enter a name");
+            return Page();
+        }
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml.cs:76]: Name and Email is optional, but is mutually inclusive if one is entered.

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 10]
```diff
@@ -0,0 +1,122 @@
...
+                <div class="gem-c-fieldset govuk-form-group">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">If you want a reply (optional)</legend>
+
+                        <p class="govuk-body">If you'd like us to get back to you, please leave your details below.</p>
+
+                        <div class="govuk-form-group">
+                            <label for="name" class="gem-c-label govuk-label">Your name</label>
+
+                            <input autocomplete="name" class="gem-c-input govuk-input" id="@nameof(Model.Name)" name="@nameof(Model.Name)" spellcheck="false" type="text" value="@(Model.Name)">
```
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:100]: Done

File: Services/CO.CDP.Organisation.WebApi/appsettings.json [hunk 11]
```diff
@@ -20,7 +20,8 @@
     "BuyerApprovedEmailTemplateId": "ebb6702c-5c08-4ed1-a611-18311f5072ee",
     "RequestToJoinNotifyOrgAdminTemplateId": "31cde325-d6a3-46b5-9d82-d2c5c43b83b1",
     "RequestToJoinConfirmationEmailTemplateId": "9c146076-f13e-4e4c-855d-d3abf25e0b87",
-    "SupportAdminEmailAddress": ""
+    "ProvideFeedbackAndContactEmailTemplateId": "51ca2e0d-aa31-4d34-a20b-d625ba33a8f8",
+    "SupportAdminEmailAddress": "CO-Support@goaco.com"
```
Comment (jakzal) [inline Services/CO.CDP.Organisation.WebApi/appsettings.json:24]: We should not expose the email.

Comment (dbgoaco) [reply Services/CO.CDP.Organisation.WebApi/appsettings.json]: Removed.

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 12]
```diff
@@ -0,0 +1,122 @@
...
+                    <p class="govuk-body">You can ask a question, report a problem or suggest an improvement to the Find a Tender team.</p>
+                }
+
+                <div class="govuk-form-group govuk-!-margin-bottom-6">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+                            <span class="govuk-fieldset__heading">What's it to do with?</span>
+                        </legend>
+
+                        <div class="govuk-radios" data-module="govuk-radios" data-govuk-radios-module-started="true">
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:45]: What does `data-govuk-radios-module-started="true"` do? I don't see it documented on the govuk design system, so unless you know what it does I think we should remove this.

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 13]
```diff
@@ -0,0 +1,122 @@
...
+
+                            <div class="govuk-radios__item">
+                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)-1" value="SpecificPage" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "SpecificPage" ? "checked" : "") aria-controls="conditional-SpecificPage" aria-expanded="true">
+                                <label for="location-1" class="govuk-label govuk-radios__label">A specific page</label>
+                            </div>
+
+                            <div class="govuk-radios__conditional govuk-body govuk-radios__conditional--hidden" id="conditional-SpecificPage">
+                                <div class="govuk-form-group">
+                                    <label for="link" class="govuk-label">Enter URL or name of page</label>
+                                    <input class="govuk-input" data-ga4-form-include-input="" id="@nameof(Model.UrlOfPage)" name="@nameof(Model.UrlOfPage)" spellcheck="false" type="text" value="@Model.UrlOfPage">
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:59]: This label is not associated with the corresponding input
```suggestion
                                    <label for="@nameof(Model.UrlOfPage)" class="govuk-label">Enter URL or name of page</label>
                                    <input class="govuk-input" id="@nameof(Model.UrlOfPage)" name="@nameof(Model.UrlOfPage)" spellcheck="false" type="text" value="@Model.UrlOfPage">
```

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 14]
```diff
@@ -0,0 +1,122 @@
...
+
+                        <div class="govuk-radios" data-module="govuk-radios" data-govuk-radios-module-started="true">
+                            <div class="govuk-radios__item">
+                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)" value="WholeSite" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "WholeSite" ? "checked" : "")>
+                                <label for="location-0" class="govuk-label govuk-radios__label">The whole site</label>
+                            </div>
+
+                            <div class="govuk-radios__item">
+                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)-1" value="SpecificPage" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "SpecificPage" ? "checked" : "") aria-controls="conditional-SpecificPage" aria-expanded="true">
+                                <label for="location-1" class="govuk-label govuk-radios__label">A specific page</label>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:53]: This label is not associated with the corresponding input
```suggestion
                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)-1" value="SpecificPage" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "SpecificPage" ? "checked" : "") aria-controls="conditional-SpecificPage" aria-expanded="true">
                                <label for="@nameof(Model.FeedbackOrContactOption)-1" class="govuk-label govuk-radios__label">A specific page</label>
```

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 15]
```diff
@@ -0,0 +1,122 @@
...
+                <div class="govuk-form-group govuk-!-margin-bottom-6">
+                    <fieldset class="govuk-fieldset">
+                        <legend class="govuk-fieldset__legend govuk-fieldset__legend--m">
+                            <span class="govuk-fieldset__heading">What's it to do with?</span>
+                        </legend>
+
+                        <div class="govuk-radios" data-module="govuk-radios" data-govuk-radios-module-started="true">
+                            <div class="govuk-radios__item">
+                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)" value="WholeSite" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "WholeSite" ? "checked" : "")>
+                                <label for="location-0" class="govuk-label govuk-radios__label">The whole site</label>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:48]: This label is not associated with the corresponding input
```suggestion
                                <input type="radio" name="@nameof(Model.FeedbackOrContactOption)" id="@nameof(Model.FeedbackOrContactOption)" value="WholeSite" class="govuk-radios__input" @(Model.FeedbackOrContactOption == "WholeSite" ? "checked" : "")>
                                <label for="@nameof(Model.FeedbackOrContactOption)" class="govuk-label govuk-radios__label">The whole site</label>
```

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml [hunk 16]
```diff
@@ -0,0 +1,122 @@
...
+                        {
+                            <label class="govuk-label govuk-label--l" for="@nameof(Model.Details)">
+                                What are the details?
+                            </label>
+                        }
+                    </h1>
+                    <div id="@nameof(Model.Details)-hint" class="govuk-hint">
+                        Do not include personal or financial information like your National Insurance number or credit card details
+                    </div>
+                    <textarea class="govuk-textarea govuk-js-character-count" id="@nameof(Model.Details)" name="@nameof(Model.Details)" rows="5">@Model.Details</textarea>
```
Comment (andymantell) [inline Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:85]: The aria-describedby attribute seems to have been removed? It should be set as follows:
```suggestion
                    <textarea class="govuk-textarea govuk-js-character-count" id="@nameof(Model.Details)" name="@nameof(Model.Details)" rows="5" aria-describedby="@nameof(Model.Details)-hint @nameof(Model.Details)-info">@Model.Details</textarea>
```

**Review by andymantell — CHANGES_REQUESTED** [review]

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml (hunk 12 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:45]: done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml (hunk 13 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:59]: done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml (hunk 14 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:53]: done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml (hunk 15 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:48]: done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml (hunk 16 above)
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml:85]: done

File: Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml (hunk 12 above)
[the remark this answers is not in the document]
Comment (dbgoaco) [reply Frontend/CO.CDP.OrganisationApp/Pages/ProvideFeedbackAndContact.cshtml]: done

**Review by andymantell — APPROVED** [review]

**Pull request merged**